pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -69,11 +221,54 @@ func GetBootstrapSecurityContextConstraints(buildControllerUsername string) []ka
 				// by the admission controller.  Admission will first look for the SupplementalGroupsAnnotation
 				// on the namespace and if it is unable to find that annotation it will attempt
 				// to use the UIDRangeAnnotation.  If neither annotation exists then creation
-				// of the SCC will fail.
+				// of the strategy will fail.
 				Type: kapi.SupplementalGroupsStrategyMustRunAs,
 			},
-			Groups: []string{AuthenticatedGroup},
 		},
+		// SecurityContextConstraintsAnyUID allows no host access and allocates SELinux.
```

Comment:
I thought this was the one we wanted admins to default to... selinux, but not forcing a uid